crates/zkforge/bin/cmd/zksolc_manager.rs

```diff
@@ -80,6 +84,9 @@ fn parse_version(version: &str) -> Result<ZkSolcVersion> {
         "v1.3.9" => Ok(ZkSolcVersion::V139),
         "v1.3.10" => Ok(ZkSolcVersion::V1310),
         "v1.3.11" => Ok(ZkSolcVersion::V1311),
+        "v1.3.13" => Ok(ZkSolcVersion::V1313),
```

Comment:
What about version `v1.3.12`?

Reply:
there is none 🤷 

https://github.com/matter-labs/zksolc-bin/releases/